cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/BaseController.java

```diff
@@ -31,6 +31,32 @@
         UpdateController.class})
 public class BaseController implements Callable<Integer> {
 
+    private static class LoggingOptions {
+        @CommandLine.Option(names = {"-q", "--quiet"}, description = "Reduce log verbosity and print errors only.")
+        private boolean quiet;
+
+        @CommandLine.Option(names = {"-v", "--verbose"}, description = "Enable verbose logging.")
+        private boolean verbose;
+
+        @CommandLine.Option(names = {"-d", "--debug"}, description = "Enable debug logging.")
+        private boolean debug;
+
+        public boolean isVerbose() {
+            return verbose;
+        }
+
+        public boolean isDebug() {
+            return debug;
+        }
+
+        public boolean isQuiet() {
+            return quiet;
+        }
+    }
+
+    @CommandLine.ArgGroup(exclusive = true, multiplicity = "0..1")
+    LoggingOptions loggingOptions;
```

Comment:
You can use a static import like: 

import static picocli.CommandLine.ArgGroup; 

then you can just write: 
ArgGroup(exclusive = true, multiplicity = "0..1")